src/browser_action.js

```diff
@@ -425,3 +425,10 @@ browseraction.createEventDiv_ = function(event) {
 window.addEventListener('load', function() {
   browseraction.initialize();
 }, false);
+
+window.addEventListener('keydown', function(event) {
+  if (event.ctrlKey && event.keyCode == 13 && $('#quick_add_button').is(':visible') && $('#quick-add-event-title').val() !== '') {
+    // Ctrl-Enter pressed
+    $('#quick_add_button').click();
```

Comment:
Instead of delegating to another event listener, extract the method into a named JavaScript function, and call it from here as well as from within `$('#quick_add_button').click();`